### PR TITLE
OP-571 #ready change text on submit button to say "search"

### PR DESCRIPTION
**Pull request by emb03**
@z3cka plz merge to test, thx!

### CODE REVIEW

File: www/sites/all/modules/contrib/search_api_solr/search_api_solr.views.inc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php
+  <?php
```
Comment (z3cka) [inline www/sites/all/modules/contrib/search_api_solr/search_api_solr.views.inc]: @emb03 why?

Comment (emb03) [inline www/sites/all/modules/contrib/search_api_solr/search_api_solr.views.inc]: I exported the view in features....

> On Oct 19, 2015, at 3:34 PM, Casey Grzecka <notifications@github.com> wrote:
> 
> In www/sites/all/modules/contrib/search_api_solr/search_api_solr.views.inc <https://github.com/UCLALibrary/libraryweb-site/pull/135#discussion_r42434960>:
> 
> > @@ -1,4 +1,4 @@
> > -<?php
> > +  <?php
> @emb03 <https://github.com/emb03> why?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/UCLALibrary/libraryweb-site/pull/135/files#r42434960>.
> 

Elaine Blakeman
Drupal Developer
Digital Initiatives and Information Technology
UCLA Library

IMPORTANT LINKS

Web Content Editors Blog  <https://docs.library.ucla.edu/pages/viewrecentblogposts.action?key=WCE>

What New Content Editors Need to Know <https://docs.library.ucla.edu/display/WCE/What+New+Web+Content+Editors+(WCEs)+Need+to+Know>

Submit a Help Ticket <mailto:helpdesk@library.ucla.edu>

Google Hangout for Training <https://goo.gl/3wvYS1> (when in session)

**Comment by akohler** [comment]
@z3cka Good catch, we don't want to change contrib files - weird that a features export would do that.

**Comment by z3cka** [comment]
Yep. she is fixing now with a squash *(interactive rebase)*

**Comment by emb03** [comment]
I don't think features did it. I think it was me not knowing what my cursor was doing.


> On Oct 19, 2015, at 3:51 PM, Casey Grzecka <notifications@github.com> wrote:
> 
> Yep. she is fixing now with a squash (interactive rebase)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/UCLALibrary/libraryweb-site/pull/135#issuecomment-149369770>.
> 

Elaine Blakeman
Drupal Developer
Digital Initiatives and Information Technology
UCLA Library

IMPORTANT LINKS

Web Content Editors Blog  <https://docs.library.ucla.edu/pages/viewrecentblogposts.action?key=WCE>

What New Content Editors Need to Know <https://docs.library.ucla.edu/display/WCE/What+New+Web+Content+Editors+(WCEs)+Need+to+Know>

Submit a Help Ticket <mailto:helpdesk@library.ucla.edu>

Google Hangout for Training <https://goo.gl/3wvYS1> (when in session)

**Pull request closed without merging**